### PR TITLE
Munitions

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

Adds big naval guns that you load and shoot with

https://cdn.discordapp.com/attachments/602204840180842497/611357480915566624/2019-08-14_14-42-55.mp4

## Why It's Good For The Game

GUNS

Thanks to Karmic for the construction code

## Changelog
:cl: Kmc2000
add: Added a railgun, torpedo tubes, and torpedo construction.
/:cl:

### CODE REVIEW

**Comment by Karmic-Skink** [comment]
MaA isn't selectable on the late join screen, but is all good on the preference selection.

File: sephora/code/modules/munitions/munitions.dm [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+
+/obj/structure/munitions_trolley/proc/unload_munition(atom/movable/A)
+	vis_contents -= A
+	A.forceMove(get_turf(src))
+	A.pixel_y = initial(A.pixel_y) //Remove our offset
+	A.layer = initial(A.layer)
+	to_chat(usr, "<span class='notice'>You unload [A] from [src].</span>")
+	if(istype(A, /obj/structure/munition)) //If a munition, allow them to load other munitions onto us.
+		capacity --
+	if(contents.len)
```
Comment (Qustinnus) [inline sephora/code/modules/munitions/munitions.dm]: unepic and not necesary. for(var/atom/movable/AM in src) is better

File: sephora/code/modules/munitions/munitions.dm [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+	dat += "<a href='?src=[REF(src)];unloadall=1'>Unload All</a>"
+	var/datum/browser/popup = new(user, "munitions trolley", name, 300, 200)
+	popup.set_content(dat)
+	popup.open()
+
+/obj/structure/munitions_trolley/Topic(href, href_list)
+	if(!in_range(src, usr))
+		return
+	var/atom/whattoremove = locate(href_list["removeitem"])
+	if(whattoremove && whattoremove.loc == src)
```
Comment (Qustinnus) [inline sephora/code/modules/munitions/munitions.dm]: use ?.

File: code/modules/jobs/job_types/warden.dm [hunk 3]
```diff
@@ -35,15 +35,14 @@
 
 	belt = /obj/item/pda/warden
 	ears = /obj/item/radio/headset/headset_sec/alt
-	uniform = /obj/item/clothing/under/ship/peacekeeper
+	uniform = /obj/item/clothing/under/syndicate/tacticool
```
Comment (francinum) [inline code/modules/jobs/job_types/warden.dm]: This suit doesn't have suit sensors.

**Pull request merged**